### ISSUE TITLE
[PR #133] Structs and Interfaces to finalize security context format, #120,

### ISSUE THREAD

**Issue by orivaris** [constructorfabric/gears-rust#2180]
> 🔗 **Mirrored PR** [cyberfabric/cyberware-rust#133](https://github.com/cyberfabric/cyberware-rust/pull/133) | **Author:** striped-zebra-dev | **Opened:** 2025-12-19T10:16:46Z | **Status:** merged on 2025-12-19T16:46:41Z
> *GitHub API does not allow setting PR author or timestamps — attribution preserved here.*

---

---

**Comment by il10241024** [comment]
**codecov-commenter** commented on 2025-12-19T10:25:34Z:

---

:warning: Please install the !['codecov app svg image'](https://github.com/codecov/engineering-team/assets/152432831/e90313f4-9d3a-4b63-8b54-cfe14e7ec20d) to ensure uploads and comments are reliably processed by Codecov.

## [Codecov](https://app.codecov.io/gh/hypernetix/hyperspot/pull/133?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=None) Report
:white_check_mark: All modified and coverable lines are covered by tests.

:loudspeaker: Thoughts on this report? [Let us know!](https://github.com/codecov/feedback/issues/255)

**Comment by il10241024** [comment]
**hyphen-2025** reviewed `libs/modkit-security/src/context.rs` line 119 on 2025-12-19T12:56:14Z:

---

```suggestion
#[cfg(test)]
#[cfg_attr(coverage_nightly, coverage(off))]
```

**Comment by il10241024** [comment]
**hyphen-2025** reviewed `libs/modkit-security/src/context.rs` line 388 on 2025-12-19T12:58:42Z:

---

It seems to be already part of `test_security_context_builder_minimal`

**Comment by il10241024** [comment]
**striped-zebra-dev** reviewed `libs/modkit-security/src/context.rs` line 119 on 2025-12-19T14:59:23Z:

---

Thx, applied.

**Comment by il10241024** [comment]
**striped-zebra-dev** reviewed `libs/modkit-security/src/context.rs` line 388 on 2025-12-19T14:59:37Z:

---

Thx, removed.
